common/app/renderers/DotcomRenderingService.scala

```diff
@@ -31,6 +25,7 @@ import play.api.libs.ws.{WSClient, WSResponse}
 import play.api.mvc.Results.{InternalServerError, NotFound}
 import play.api.mvc.{RequestHeader, Result}
 import play.twirl.api.Html
+import services.NewsletterData
```

Comment:
Is this an IDE thing?

Reply:
yep Intellij just rearranged the imports